desktop/src/main/java/bisq/desktop/main/portfolio/PortfolioView.java

```diff
@@ -138,6 +143,7 @@ private void onEditOpenOfferRemoved() {
             editOfferView = null;
         }
 
+        // navigation.navigateToPreviousVisitedView();
```

Comment:
I think it would be nice to return to Offer Book here. Unfortunately this method didn't worked as expected - it moves me to Portfolio -> History. I'll try to debug why `previousPath` is calculated as `ClosedTradesView`  [here](https://github.com/bisq-network/bisq/blob/7f73df5dfd761a26392fb6ae16adc937ef6bf9da/desktop/src/main/java/bisq/desktop/Navigation.java#L111-L139) when I [click](https://github.com/xyzmaker123/bisq/blob/0ab0aed79ab2d8762be935ea77330ba693d62bad/desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBookView.java#L737-L739) edit button. Any tips in the meantime are welcomed.

Reply:
Let's skip it for now - navigation logic is too complicated for me at the moment.